lib/tasks/downloads.rake

```diff
@@ -41,19 +41,23 @@ task :downloads => [:environment, :windows_downloads, :mac_downloads]
 
 task :windows_downloads => :environment do
   # find latest windows version
-  project = "msysgit"
-  win_downloads = file_downloads_from_github("msysgit/msysgit")
+  project = "git-for-windows"
+  win_downloads = file_downloads_from_github("git-for-windows/git")
   win_downloads.each do |url, date|
     name = url.split('/').last
-    if m = /^Git-(.*?)-(.*?)(\d{4})(\d{2})(\d{2})\.exe/.match(name)
-      version = m[1]
+    if m = /^(.*?)Git-(.*?)-(.*?)-(.*?)-(.*?)-bit\.(.*?)exe/.match(name)
+      portable = m[1]
+      version = m[2]
```

Comment:
`[Portable]Git-#.#.#.#[-dev-preview]-32/64-bit[.7z].exe` is the current system. @dscho  correct me if I'm wrong. So the [-dev-preview] part is (at least in the far future) optional. So I think you are correct the current implementation is indeed wrong. Any ideads? I'm not that a big *Regex* technie